src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -95,6 +105,17 @@ public static boolean start(final RootDoc rootDoc) throws IOException {
          return new HelpDoclet().startProcessDocs(rootDoc);
      }
 
+     private void validateSettingsDir() {
+         if (!settingsDir.exists()) {
+             logger.error(SETTINGS_DIR_OPTION + " : " + settingsDir.getPath() + " does not exist - will fallback on classpath: " + DEFAULT_SETTINGS_CLASSPATH);
+             settingsDirIsValid = false;
+         }
+         else if (!settingsDir.isDirectory()) {
+             logger.error(SETTINGS_DIR_OPTION + " : " + settingsDir.getPath() + " is not a directory - will fallback on classpath: " + DEFAULT_SETTINGS_CLASSPATH);
+             settingsDirIsValid = false;
+         }
+     }
+
```

Comment:
I think we should find a more explicit way to signal the use of the classpath falback; maybe add an explicit argument that says to use the default, and then make it mutually exclusive with settings-dir (throw if they're both provided).